@xen-orchestra/web-core/lib/components/ui/card-numbers/UiCardNumbers.vue

```diff
@@ -17,34 +19,26 @@
 <script setup lang="ts" generic="TSize extends 'small' | 'medium'">
 import { toVariants } from '@core/utils/to-variants.util'
 import { computed } from 'vue'
-import { useI18n } from 'vue-i18n'
+import { I18nN } from 'vue-i18n'
 
-const props = defineProps<{
+const { label, size, value, unit, max } = defineProps<{
```

Comment:
```suggestion
const { size, value, max } = defineProps<{
```

They are not use in the `<script>`